main/src/ui/conversation_content_view/file_transmission_progress.vala

```diff
@@ -80,17 +80,14 @@ namespace Dino.Ui {
         }
 
         private void update_progress() {
-            this.get_style_context().remove_provider(css_provider);
-            css_provider = new CssProvider();
             int64 progress_percent = transferred_size * 100 / file_size;
 
-            css_provider.load_from_string(@"
+            Util.force_css(this, @"
```

Comment:
This funktion returns the created CssProvider. Please assign it to css_provider and keep the remove_provider() in, such that we don't pile one CssProvider on top of the other the whole time.